test/plugins/features/movies/controller.test.js

```diff
@@ -9,10 +11,66 @@ describe('movie controller', () => {
     it('creates a movie', async () => {
       const payload = { title: 'WALL-E' };
       const movie = await Controller.create(payload);
-
       expect(movie.get('title')).to.eql(payload.title);
     });
 
   });
 
+  describe('retrieve', () => {
+
+    beforeEach(async () => {
+      await Knex.raw('DELETE FROM movies;');
+
+      this.payload = { title: 'Zodiac', release_year: 2007 };
+      this.payload2 = { title: 'Dawn of the Planet of the Apes', release_year: 2014 };
+
+      await Controller.create(this.payload);
+      await Controller.create(this.payload2);
```

Comment:
Typically in test setup, you won't want to use a function from the component you're testing. I'd recommend using `Knex` directly to insert the expected movies. We typically would set up a `Movie` factory that we use to generate movies to insert, but in this case we could keep it simple and do something like:
```
  await Knex('movies').insert([
    {
      title: 'Zodiac',
      release_year: 2007
    },
    {
      title: 'Dawn of the Planet of the Apes',
      release_year: 2014
    }
  ]);
```

That way we're inserting these movies directly into the DB and aren't relying on `Controller.create` to do what we expect it to do.

As a side-note, you should avoid using `this` in your code unless it's actually necessary. In this case, you should define those objects as `const`'s rather than adding them to `this`.  `const`'s are block scoped, so since you reference them below, you'd just want to define them before the `beforeEach` but still within the `describe` block. Let me know if you have any questions about that!